src/ir/ExpressionAnalyzer.cpp

```diff
@@ -137,7 +137,8 @@ template<typename T> void visitImmediates(Expression* curr, T& visitor) {
       visitor.visitInt(curr->isReturn);
     }
     void visitCallIndirect(CallIndirect* curr) {
-      visitor.visitNonScopeName(curr->fullType);
+      visitor.visitInt(curr->sig.params);
+      visitor.visitInt(curr->sig.results);
```

Comment:
IIUC this hashes the int id of the params and the results? Those are globally unique and interned, correct?

I assume those values aren't deterministic in multithreaded runs, which could in theory be confusing when debugging, but then the previous hash was also nondeterministic here I guess...

Reply:
Yes, your understanding is correct here.